pinot-segment-local/src/main/java/org/apache/pinot/segment/local/indexsegment/immutable/ImmutableSegmentImpl.java

```diff
@@ -150,7 +150,7 @@ public void deleteValidDocIdsSnapshot() {
     }
   }
 
-  private File getValidDocIdsSnapshotFile() {
+  public File getValidDocIdsSnapshotFile() {
```

Comment:
No longer needed?

Reply:
thanks @Jackie-Jiang for review. since getValidDocIdsSnapshotFile() is not being called, i revert this change back.